cmd/container-manager/manage_ctr_mgr.sh

```diff
@@ -105,7 +105,11 @@ install_docker(){
 			sudo -E apt-get -y install apt-transport-https ca-certificates software-properties-common
 			repo_url="https://download.docker.com/linux/ubuntu"
 			curl -fsSL "${repo_url}/gpg" | sudo apt-key add -
-			sudo -E add-apt-repository "deb [arch=${arch}] ${repo_url} $(lsb_release -cs) stable"
+			if [ "$(echo "${VERSION_ID} >= 20.04" | bc -q)" == "1" ]; then
+				sudo -E add-apt-repository "deb [arch=${arch}] ${repo_url} bionic stable"
```

Comment:
I'm really not keen on doing things like this. Do we really *really* need these btrfs tools installed if we're not using btrfs?

Reply:
@jodh-intel, I am not sure if I understand you...should I just do the 
`${VERSION_ID} >= "20.04"`

Reply:
no @GabyCT , since we don't use `btrfs` in kata containers, maybe the package `btrfs-tools` is not longer needed

Reply:
would be good to check building/installing `containerd/cri`, as I remember I needed to add it for another distro.
@GabyCT maybe you already did, but if not, please try to run `.ci/install_cri_containerd.sh`. If that finishes successfully, we can remove the package.